src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -61,13 +61,13 @@ New-AzStorageContext [-StorageAccountName] <String> [-UseConnectedAccount] [-Pro
  -Environment <String> [<CommonParameters>]
 ```
 
-### AccountNameAndKeySeviceEndpoint
+### AccountNameAndKeyServiceEndpoint
```

Comment:
This part in help should be auto generated. Don't manually write it.
Normally it should not wrong. 

Please make sure the help file is auto generated.

Reply:
Definitely! Thanks a lot :)